packages/quantic/force-app/main/default/lwc/quanticCarousel/__tests__/quanticCarousel.test.js

```diff
@@ -205,58 +208,58 @@ describe('c-quantic-carousel', () => {
   });
 
   describe('when used with invalid properties', () => {
-    it('should display an error message when the property numberOfPages is undefined', async () => {
-      const element = createTestComponent({
-        ...defaultOptions,
-        numberOfPages: undefined,
-      });
-      await flushPromises();
+    let consoleError;
+    
+    beforeEach(() => {
+      consoleError = jest.spyOn(console, "error").mockImplementation(() => {});
```

Comment:
BTW just FYI, but you can also do [mockImplementationOnce(fn)](https://jestjs.io/docs/mock-function-api#mockfnmockimplementationoncefn) to mock it only once. 

It could be useful if you want to just mock it inside one test, and just mock it for exactly one and only one call to console.error.